packages/web-components/src/main.d.ts

```diff
@@ -15,9 +15,20 @@
  */
 
 /** Interfaces for the attributes for each web component */
-import React from "react";
+export type SORT_ASCENDING = "ascending";
+export type SORT_DESCENDING = "descending";
+export type SORT_ASCENDING_POPULATION = "ascendingPopulation";
+export type SORT_DESCENDING_POPULATION = "descendingPopulation";
+export type ChartSortOption =
+  | SORT_ASCENDING
+  | SORT_DESCENDING
+  | SORT_ASCENDING_POPULATION
+  | SORT_DESCENDING_POPULATION;
 
-import { SortType } from "../js/chart/types";
+export interface ChartEventDetail {
```

Comment:
nit: can we add a quick comment about what this is for?

Reply:
done!